python/src/deltachat/account.py

```diff
@@ -190,36 +191,36 @@ def set_avatar(self, img_path):
             assert os.path.exists(img_path), img_path
             self.set_config("selfavatar", img_path)
 
-    def check_is_configured(self):
+    def check_is_configured(self) -> None:
         """ Raise ValueError if this account is not configured. """
         if not self.is_configured():
             raise ValueError("need to configure first")
 
-    def get_latest_backupfile(self, backupdir):
+    def get_latest_backupfile(self, backupdir) -> Optional[str]:
         """ return the latest backup file in a given directory.
         """
         res = lib.dc_imex_has_backup(self._dc_context, as_dc_charpointer(backupdir))
         if res == ffi.NULL:
             return None
         return from_dc_charpointer(res)
 
-    def get_blobdir(self):
+    def get_blobdir(self) -> Optional[str]:
         """ return the directory for files.
 
         All sent files are copied to this directory if necessary.
         Place files there directly to avoid copying.
         """
         return from_dc_charpointer(lib.dc_get_blobdir(self._dc_context))
 
-    def get_self_contact(self):
+    def get_self_contact(self) -> Contact:
         """ return this account's identity as a :class:`deltachat.contact.Contact`.
 
         :returns: :class:`deltachat.contact.Contact`
         """
         return Contact(self, const.DC_CONTACT_ID_SELF)
 
-    def create_contact(self, obj, name=None):
-        """ create a (new) Contact or return an existing one.
+    def create_contact(self, obj, name: Optional[str] = None) -> Contact:
```

Comment:
```suggestion
    def create_contact(self, obj: Union[Account, Contact, str], name: Optional[str] = None) -> Contact:
```

Reply:
`flake8` says "undefined name 'Account'", I found no way to fix it.